Prsfixed

Amelia0010 opened the pull request:
all of those PRs.

ShadowDeath6 commented:
why does the youtool need a coin that should be used on (redacted) for stuff that you can get normally

also you could just put the youtool on the tile directly below the vendomat instead of moving the tables

also also, there's a bugged light on top of the middle table, not a huge fan of the tables being moved either, it makes random empty space on the right side of the room and constrains space in the middle where people should have room

Yewmi commented:
Nobody seems to care for this anyway.

silicons committed:
Deconstruction for anomalies in techwebs (#37912)

SpaceManiac committed:
Fix some areas near Pubby's starboard solars (#38001)

Commit:
Buffs atmospheric analyzers to show mole count, volume and higher degrees of precision (#37631)

add: Analyzers can now scan all kinds of atmospheric machinery - unary, binary, ternary, quaternary, you name it. This means stuff like gas pumps, gas mixers, vents and so forth can be analyzed.

tweak: Analyzers now show temperature in kelvin as well as celsius.

tweak: Analyzers now show total mole count, volume, and mole count of all gases.

tweak: Analyzers show everything at slightly higher degrees of precision.


More accurate and advanced analyzers have been something that I think atmos and toxins mains have been wanting for a while now (including myself). It's always a pain to have to manually add 273.15 to every celsius temperature, or have to use outside tools or VV (which isnt usable in actual games) to calculate mole count for a gas mixture. Not to mention, the ability to analyze things like volume pumps and gas filters makes it easier to manage larger atmos setups. This is just a QoL change for people in any of those jobs. In fact, most players probably don't even know what the fuck is a mole is so adding mole readout to analyzers is a good way to get people to understand atmospherics better.

imsxz committed:
Add logging to mayhem bottle and blood frenzy (#37951)

* who the fuck didnt make this logged ages ago jesus fucking christ

* who fucking coded this garbage why isnt it a status effect

* Update mines.dm

* reeeeeeeee

* reeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee

ShizCalev committed:
Fixes gulag teleporter runtime (#37962)

optimumtact committed:
Merge pull request #37984 from AutomaticFrenzy/patch/thunderdome

Fix entertainment monitors viewing the wrong Thunderdome

SpaceManiac committed:
Fix flipping of trinary pipe fittings and related RPD tooltips (#37982)

SpaceManiac committed:
Fix outdated R&D console comment (#38020)

xenon-difluoride committed:
Fixes mimery blockade showing up under spell tab (#38004)

KorPhaeron committed:
Fixes reactive repulse armour (#37999)

vuonojenmustaturska committed:
Removes a couple thousand empty lists (#38000)

door.shockedby

MrDoomBringer committed:
Adds a sprite for the "quit vr" action button (#37994)

* is that a motherfucking 90s reference

* WHEN YA FUCK UP BADLY ENOUGH ON ADDING A SPRITE TGHAT YA GOTTA MANUALLY CHANGE CHANGELOG.HTML

Nabski89 committed:
Magical Animations Update Part 2: Staffs (#37930)

* Add files via upload

* WHY IS THIS SO HARD

* fuck books

Garen7 committed:
Some circuitry fixes and qol (#37755)

* some circuit fixes

I removed the ability for you to save whether or not an assembly is open since you always want the assebly to be open when its printed so you can put a battery in. I fixed the other three issues because they were bugs.

* tells user when they use a sensor or scanner

* oops

* revert last commit

* usr -> user

ShizCalev committed:
[s] Power monitor tweaks and fixes (#38005)

* Power monitor tweaks and fixes

* Crouching tiger, hidden power monitor

* flavor text

SpaceManiac committed:
Fix ambient occlusion on xeno weeds and wall corners (#37987)

Amelia0010 committed:
tesla_act refactors and balancing (#37369)

experimental: Tesla damage has been reworked to structures and objects have been worked with a linear scaling.

experimental: All tesla weapons now no longer need to hit a mob to arc lightning; however, they will no longer stun, and only hit each target once.

tesla_zap vars are now a single bitfield.

Amelia0010 committed:
atom.initialized etc

Amelia0010 committed:
Merge branch 'master' into prsfixed
